base/precompilation.jl

```diff
@@ -439,14 +439,50 @@ function precompilepkgs(pkgs::Vector{String}=String[];
     append!(project_deps, keys(filter(d->last(d).name in keys(env.project_deps), ext_to_parent)))
 
     @debug "precompile: deps collected"
+
+    # An extension effectively depends on another extension if it has all the the
```

Comment:
This comment is no longer true I guess (since https://github.com/JuliaLang/julia/pull/55589)?

Reply:
Updated